core/tests/protractor_utils/users.js

```diff
@@ -96,17 +102,15 @@ var _completeSignup = async function(username, manualNavigation = true) {
   var agreeToTermsCheckbox = element(
     by.css('.protractor-test-agree-to-terms-checkbox'));
   var registerUser = element(by.css('.protractor-test-register-user'));
-  await waitFor.visibilityOf(
-    usernameInput, 'No username input field was displayed');
-  await usernameInput.sendKeys(username);
-  await agreeToTermsCheckbox.click();
-  await registerUser.click();
+  await action.sendKeys('Username input', usernameInput, username);
+  await action.click('agreeToTerms Checkbox', agreeToTermsCheckbox);
+  await action.click('Register User button', registerUser);
   await waitFor.pageToFullyLoad();
 };
 
 var completeLoginFlowFromStoryViewerPage = async function(email, username) {
-  await login(email, false, false);
-  await _completeSignup(username, false);
+  await login(email, true);
+  await _completeSignup(username);
```

Comment:
Are these expected changes?

Reply:
If I don't set manualNavigation to true, Following happens:
![Screenshot from 2021-02-13 11-57-08](https://user-images.githubusercontent.com/24855641/107843627-d64fcf80-6df2-11eb-865a-8e4ff59a9c35.png)
![Screenshot from 2021-02-13 11-57-58](https://user-images.githubusercontent.com/24855641/107843628-d8b22980-6df2-11eb-9d11-184e3195cba6.png)
for this test:
![Screenshot from 2021-02-13 11-59-06](https://user-images.githubusercontent.com/24855641/107843633-e667af00-6df2-11eb-8386-1f83ec0a812b.png)

isSuperAdmin should be false, that was a mistake, It will change it.